frontend/src/app/components/search/search.component.ts

```diff
@@ -106,6 +107,7 @@ export class SearchComponent implements OnInit {
       this.years = year.filter(this.onlyUnique).sort();
 
       this.loading = false;
+      setTimeout(_=>{this.searchBox.focus();});
```

Comment:
The original setTimeout was needed in ngAfterViewInit(), it seems like its not needed when used after `loading` is set, I assume it to be a timing bug. But however, assuming that loading n/w resource will take longer time than rendering, I'll remove `setTimeout` altogether.